### ISSUE TITLE
Introduce configuration via the command line

### ISSUE THREAD

**Issue by dogweather** [dogweather/analyze-ors-amendment-haskell#1]
[Tika.hs](https://github.com/public-law/analyze-oregon-law-haskell/blob/master/analyze/src/Tika.hs) has a hard-coded location for the Tika java library:

```Haskell
jarFile        = "/Users/robb/lib/tika-app.jar"
```

Let's change this to be configured via a command line argument.

**Comment by astynax** [comment]
I'll do this.

**Comment by dogweather** [comment]
Awesome, thank you.

**Comment by joehillen** [comment]
@astynax Oops! Sorry! I didn't realize it was a race. :wink:

**Comment by joehillen** [comment]
@dogweather Neat project! I saw this on the hacktoberst list via Reddit, and I love when code meets law so I thought I'd contribute.

I recommend using java's `CLASSPATH` environment variable instead, like this:

```
java -cp '/home/joe/Downloads/tika-app-1.16.jar' org.apache.tika.cli.TikaCLI
```

I opened the PR to add that, but a proper CLI is of course not a bad idea.

Some future work suggestions:

- I recommend using Elasticsearch as a backend. It could be the interface between your webapp and this analyzer rather than using JSON via stdout. It would give you much more power to dig through the data. I imagine Kibana would also be handy.
- I might be better to use [inline-java](https://www.tweag.io/posts/2016-10-17-inline-java.html)  to read Tika's internal representation rather than parse its HTML output.
- Although they're harder to write, parses are much more powerful than regex. I'd rewrite the parsing to use [megaparsec](https://hackage.haskell.org/package/megaparsec).
